src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -160,10 +160,7 @@ std::vector<std::string> disabledTestPatterns() {
         R"(.*CompileModelCacheTestBase.*CompareWithRefImpl.*Nms.*)",
         // Issue: 76980
         R"(.*smoke_Auto_BehaviorTests.*InferDynamicNetwork/.*)",
-        // enable after other plugins support nms9 as setup with nms5 in
-        // tests/functional/shared_test_classes/include/shared_test_classes/single_layer/non_max_suppression.hpp
-        // is shared across plugins
-        // passed local test and cpu has specific test cases with nms9 to cover
+        // Issue: 105838
```

Comment:
@chenhu-wang @iefode unfortunately there would be 16 test failures with smoke_NmsLayerTest in current master branch (without changes in this PR). I plan to address this legacy issue in a separate bug-fixing PR. 

PR16128 could be covered by local test suite - smoke_NmsLayerCPUTest.